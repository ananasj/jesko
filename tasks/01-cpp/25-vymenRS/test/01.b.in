4 4
4 2 1 6
3 1 8 3
7 4 2 1 
9 1 3 2
