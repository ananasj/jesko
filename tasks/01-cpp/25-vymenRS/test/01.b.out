4 3 7 9 
2 1 4 1 
1 8 2 3 
6 3 1 2 
